program/templates/program/talk.html

```diff
@@ -79,20 +80,29 @@ <h2>About {{ speaker.name }}</h2>
     </div>
   {% endif %}
 
+  {% if speaker.mastodon %}
+    <div class="mastodon">
+      <a href="https://{{ speaker.mastodon }}" target="_blank">
```

Comment:
```suggestion
      <a href="https://{{ speaker.mastodon | remove_scheme_prefix }}" target="_blank">
```

maybe here too if we're going to as for `full URL` (just in case)?